Sources/Tracking/CustomerIO.swift

```diff
@@ -173,6 +112,38 @@ public class CustomerIO: CustomerIOInstance {
         }
     }
 
+    /**
+     Initialize the shared `instance` of `CustomerIO` with optional `configParams`.
+     Call this function when your app launches, before using `CustomerIO.instance`.
+     */
+    @available(iOSApplicationExtension, unavailable)
+    public static func initialize(
+        siteId: String,
+        apiKey: String,
+        region: Region,
+        params: [String: Any] = [:],
+        configure configureHandler: ((inout SdkConfig) -> Void)?
```

Comment:
What is the difference between `params` and `configure`? As I understand from the comment above, `param` is an optional configuration but for what ? When configure expects the same kind of configuration ? 🤔

Reply:
`params` are pre-provided SDKConfig initializing parameters, they just take away the responsibility of parsing from clients to native SDK. 

While `configure` expects clients to give their configuration. 

And in order to reduce the number of initializer methods, we combined them.

Reply:
Gotcha. I think it would be helpful to update the comment specifying it's usage only from wrappers, since it is a public function, our users will have access to it too.

Reply:
Added